lucille-core/src/test/java/com/kmwllc/lucille/stage/ComputeFieldSizeTest.java

```diff
@@ -0,0 +1,79 @@
+package com.kmwllc.lucille.stage;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import java.util.Arrays;
+import org.junit.Test;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.kmwllc.lucille.core.UpdateMode;
+
+public class ComputeFieldSizeTest {
+
+  private final StageFactory factory = StageFactory.of(ComputeFieldSize.class);
+
+  @Test
+  public void testNotAllRequiredFields() throws StageException {
+    assertThrows(StageException.class, () -> {
+      factory.get("ComputeFieldSizeTest/no-source.conf");
+    });
+    assertThrows(StageException.class, () -> {
+      factory.get("ComputeFieldSizeTest/no-destination.conf");
+    });
+  }
+
+  @Test
+  public void testSourceFieldDoesNotExist() throws StageException {
```

Comment:
In general we want stages to be tolerant of a source field not existing and simply skip processing in that case. Lots of different kinds of documents could be flowing through a lucille pipeline and the goal is to "get as many documents through" as possible.  You can see this approach in stages like CopyFields and DeleteFields, there's a conditional like 

      if (!doc.has(sourceField)) {
        continue;
      }

Let's take the same approach here.